### ISSUE TITLE
tab bar bug!

### ISSUE THREAD

**Issue by charlesluo2014** [aksonov/react-native-router-flux#1233]
### Version

Tell us which versions you are using: 
- react-native-router-flux v3.35.0
- react-native v0.3.4
### Expected behaviour

Tab bar show items,and click one item will show all tabs,and selected
### Actual behaviour

click one tab,and the other tabs disapear!
### Steps to reproduce

1.run example
2.click the "Go to tab page"
3.click first tab

**Comment by charlesluo2014** [comment]
this bug is in the Reducer.js,the old version is:

```
case ActionConst.JUMP:
      assert(state.tabs, `Parent=${state.key} is not tab bar, jump action is not valid`);
      ind = -1;
      state.children.forEach((c, i) => { if (c.sceneKey === action.key) { ind = i; } });
      assert(ind !== -1, `Cannot find route with key=${action.key} for parent=${state.key}`);
      return { ...state, index: ind };
```
and the new version:
```
case ActionConst.JUMP:
      assert(state.tabs, `Parent=${state.key} is not tab bar, jump action is not valid`);
      ind = -1;
      children = getInitialState(props, scenes, ind, action);
      children = Array.isArray(children) ? children : [children];
      children.forEach((child, i) => {
        if (child.sceneKey === action.key) ind = i;
      });

      assert(ind !== -1, `Cannot find route with key=${action.key} for parent=${state.key}`);

      if (action.unmountScenes) {
        resetHistoryStack(state.children[ind]);
      }
      return { ...state, index: ind, children };
```

**Comment by aksonov** [comment]
@charlesluo2014 Please check latest master now.

**Comment by alizbazar** [comment]
This is fixed in 3.36.0. Had the same issue.
